tests/test_plotting.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+
+import camelot
+import os
+import pytest
+
+import matplotlib.pyplot as plt
+
+from camelot.plotting import *
+
+
+testdir = os.path.dirname(os.path.abspath(__file__))
+testdir = os.path.join(testdir, "files")
+
+
+@pytest.mark.mpl_image_compare(baseline_dir="files/baseline_plots")
+def test_text_plot():
+    filename = os.path.join(testdir, "foo.pdf")
+    tables = camelot.read_pdf(filename)
+    text = tables[0]._text
+    fig = plot_text(text)
+    ax = fig.axes[0]
+    xs, ys = [], []
+    for t in text:
+        xs.extend([t[0], t[2]])
+        ys.extend([t[1], t[3]])
+    assert ax.get_xlim() == (min(xs) - 10, max(xs) + 10)

```

Comment:
@Suyash458 Why do we need additional asserts here? Doesn't pytest compare the whole image which is a sure shot way of knowing whether the images are the same or not?

Reply:
Yes and in the .travis.yml too. After that we shouldn't need to worry about additional asserts.